db/migrate/20250420031735_migrate_organization_partner_form_fields_executive_director_to_contacts.rb

```diff
@@ -0,0 +1,27 @@
+class MigrateOrganizationPartnerFormFieldsExecutiveDirectorToContacts < ActiveRecord::Migration[8.0]
+  class Organization < ApplicationRecord
```

Comment:
Unlikely the Organization name would change, but this is a way to allow us to use the model name in the migration.